src/worker/link-storage-to-kmp.ts

```diff
@@ -8,10 +8,15 @@ export async function linkStorageToKmp(storage: Storage): Promise<ArrayBuffer> {
     storage
   );
 
+  const maybePackageInfo = await storage.fetchPackageInfo();
+  if (maybePackageInfo == undefined) {
+    throw new Error("the packageInfo is undefined");
```

Comment:
I know we wrote this together, but a better error message would indicate what exactly is wrong, and why it might be wrong:

```suggestion
    throw new Error("could not fetch the package info; did you update the package BCP-47 tag?");
```